unlock-app/src/components/content/MembersContent.tsx

```diff
@@ -166,12 +167,15 @@ const MetadataTableWrapper = ({
   const [currentPage, setCurrentPage] = useState(page)
   const [query, setQuery] = useState<string>('')
   const [filterKey, setFilteKey] = useState<string>('owner')
+  const [currentFilter, setCurrentFilter] = useState<Filter>()
   const queryValue = useDebounce<string>(query)
 
-  const filters: Filter[] = [
-    { key: 'owner', label: 'Owner' },
-    { key: 'keyId', label: 'Token id' },
-  ]
+  const filters: Filter[] = useMemo(() => {
```

Comment:
Why `useMemo` ?

Reply:
same ^ 

it's unnecessary given the negligible cost of creating an object in js. In fact, it might be slower due to the fact it does a lot in the background. Just put this static variable outside the component if you don't want to recreate it on component mount/unmount.